### PR TITLE
feat(Apps): Adds Sales Demo Data App. (LL-103)

### PR DESCRIPTION
**Pull request by lydiamross**
Please remember to request a review from someone when this PR is ready for a code review.

#### Which issue does this close?
Open source version of:
[LL-103](https://learningpool.atlassian.net/browse/LL-103)

### CODE REVIEW

File: ui/src/pages/SettingsAppsPage/RequestAppAccessForm.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  connect(
+    state => ({
+      requestState: requestStateSelector()(state),
+    }),
+    { requestAppAccess }
+    ),
+  formDisplayState,
+  privacyPolicyState,
+  userMessageState,
+  handlers,
```
Comment (ryansmith94) [inline ui/src/pages/SettingsAppsPage/RequestAppAccessForm.js]: Let's convert these to hooks instead, remind me to help you with that if you need some help doing that.

File: ui/src/utils/LLApiClient.js [hunk 2]
```diff
@@ -96,6 +96,15 @@ class _LLApiClient {
     }
   });
 
+  requestAppAccess = ({ privacyPolicy, appName }) => post({
+    url: formatUrl(routes.REQUEST_APP_ACCESS),
+    headers: {
+      Authorization: `Bearer ${this.getToken()}`,
+      ContentType: 'application/json'
+    },
+    body: { privacyPolicy, appName },
```
Comment (ryansmith94) [inline ui/src/utils/LLApiClient.js]: It's possible that Enterprise may require more info than this for other apps.

File: api/src/controllers/RequestAppAccessController.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
```
Comment (ryansmith94) [inline api/src/controllers/RequestAppAccessController.js]: Would prefer that we moved towards individual files for each request handler, but that change hasn't been merged into master so don't worry about it here, just something to bear in mind in the future once [LL-228](https://learningpool.atlassian.net/browse/LL-228) is merged.

File: ui/src/utils/LLApiClient.js (hunk 2 above)
Comment (lydiamross) [reply ui/src/utils/LLApiClient.js]: I could change to sending an object containing that info? but not sure how to format the email template if doing that.

File: api/src/controllers/RequestAppAccessController.js (hunk 3 above)
Comment (lydiamross) [reply api/src/controllers/RequestAppAccessController.js]: ok thanks

File: ui/src/utils/LLApiClient.js (hunk 2 above)
Comment (ryansmith94) [reply ui/src/utils/LLApiClient.js]: Note to self: Discussed in person and decided to send config as JSON in email.

**Pull request merged**